platform/fabric/services/state/options.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
why is this part of the PR?

Reply:
I needed this for the integration test to be able to choose a particular identity at time of exchanging identitites.